bears/ruby/RubySmellBear.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange
+
+
+@linter(executable='reek',
+        use_stdin=True)
+class RubySmellBear:
+    """
+    Detect code smells in Ruby source code.
+    """
+
+    LANGUAGES = "Ruby"
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return '--format', 'json'
+
+    def process_output(self, output, filename, file):
+        output = json.loads(output) if output else []
```

Comment:
if it hangs there will usually be no json output because you usually first create the results and then output then, solely from a technical perspective